### PR TITLE
Get latest data from world bank data for country overview

### PR DESCRIPTION
**Pull request by Rup-Narayan-Rajbanshi**
Addresses
- https://github.com/IFRCGo/go-web-app/issues/1281
- https://github.com/IFRCGo/go-api/issues/2224

## Changes
- Add latest data from World Bank API for country overview.
- Add [colorlog](https://github.com/borntyping/python-colorlog) for development console
- Add helper for logger context

## Checklist
Things that should succeed before merging.

- [ ] Updated/ran unit tests
- [ ] Updated CHANGELOG.md

### CODE REVIEW

**Comment by Rup-Narayan-Rajbanshi** [comment]
This PR is under progress so, do not review comment now.

**Review by thenav56 — APPROVED** [review]
@Rup-Narayan-Rajbanshi Let's add some notes on refactor required

```
XXX: Using char to store integer values, Need to refactor the column type
```

**Review by susilnem — APPROVED** [review]
LGTM!

**Review by thenav56 — APPROVED** [review]

**Comment by samshara** [comment]
@Rup-Narayan-Rajbanshi  It seems we haven't update the year of latest data
```"world_bank_population": 42239854,
    "calculated_world_bank_population_year": null,
    "world_bank_population_above_age_65": 1011704,
    "calculated_world_bank_population_above_age_65_year": null,
    "world_bank_population_age_14": 18097715,
    "calculated_world_bank_population_age_14_year": null,
    "world_bank_urban_population_percentage": 26.933,
    "calculated_world_bank_urban_population_percentage_year": null,
    "world_bank_gdp": 14502158192.0904,
    "calculated_world_bank_gdp_year": null,
    "world_bank_gni": 14545219074.5763,
    "calculated_world_bank_gni_year": null,
    "world_bank_gender_inequality_index": 1.0,
    "calculated_world_bank_gender_inequality_index_year": null,
    "world_bank_life_expectancy": 62,
    "calculated_world_bank_life_expectancy_year": null,
    "world_bank_literacy_rate": 37.266040802002,
    "calculated_world_bank_literacy_rate_year": null,
    "world_bank_poverty_rate": 54.5,
    "calculated_world_bank_poverty_rate_year": null,
    "world_bank_gni_capita": 360,
    "calculated_world_bank_gni_capita_year": null
    ```

**Comment by Rup-Narayan-Rajbanshi** [comment]
> @Rup-Narayan-Rajbanshi It seems we haven't update the year of latest data
> 
> ```
>     "world_bank_population": 42239854,
>     "calculated_world_bank_population_year": null,
>     "world_bank_population_above_age_65": 1011704,
>     "calculated_world_bank_population_above_age_65_year": null,
>     "world_bank_population_age_14": 18097715,
>     "calculated_world_bank_population_age_14_year": null,
>     "world_bank_urban_population_percentage": 26.933,
>     "calculated_world_bank_urban_population_percentage_year": null,
>     "world_bank_gdp": 14502158192.0904,
>     "calculated_world_bank_gdp_year": null,
>     "world_bank_gni": 14545219074.5763,
>     "calculated_world_bank_gni_year": null,
>     "world_bank_gender_inequality_index": 1.0,
>     "calculated_world_bank_gender_inequality_index_year": null,
>     "world_bank_life_expectancy": 62,
>     "calculated_world_bank_life_expectancy_year": null,
>     "world_bank_literacy_rate": 37.266040802002,
>     "calculated_world_bank_literacy_rate_year": null,
>     "world_bank_poverty_rate": 54.5,
>     "calculated_world_bank_poverty_rate_year": null,
>     "world_bank_gni_capita": 360,
>     "calculated_world_bank_gni_capita_year": null,
> ```

I did run ingest_worldbank mgmt task. Now it working fine.

**Review by susilnem — APPROVED** [review]

File: databank/management/commands/ingest_worldbank.py [hunk 1]
```diff
@@ -1,177 +1,224 @@
...
+            .exclude(iso3__in=["COK", "BAR", "NOR"])
         )
 
-        world_bank_indicators = [indicator for indicator, _ in world_bank_indicator_map]
-        country_dict = {}
+        total_countries = country_qs.count()
+        for index, country in enumerate(country_qs, start=1):
+            iso3 = country.iso3
+            self.stdout.write(f"Importing country ({index:03}/{total_countries}): {iso3}")
+            overview = country.countryoverview
```
[the remark this answers is not in the document]
Comment (thenav56) [reply databank/management/commands/ingest_worldbank.py:172]: Let's do that, we can do that easily with
```python
overview = CountryOverview(id=country.id)
```
> NOTE: CountryOverview primary key is country id

**Review by thenav56 — APPROVED** [review]

**Comment by szabozoltan69** [comment]
@Rup-Narayan-Rajbanshi should we merge it to develop (and deploy to Staging)?

**Comment by szabozoltan69** [comment]
@samshara  should we merge it to develop (and deploy to Staging)?

**Comment by szabozoltan69** [comment]
@thenav56 – Done, also the deployment to Staging.

**Comment by szabozoltan69** [comment]
```
Running migrations:
  Applying databank.0026_countryoverview_calculated_world_bank_gdp_year_and_more... OK
```

**Pull request merged**